Workspace/index.html

```diff
@@ -7,6 +7,6 @@
   </head>
   <body>
     Hello world, My name is weyinmi Eyoyibo and i am an aspiring front-end
-    developer
+    developer My goal is to become a seasoned front-end web developer.

```

Comment:
add a full spot (.) after the developer before the 'My'

Reply:
Done

Reply:
i have now